simulation/g4main/HepMCNodeReader.h

```diff
@@ -59,6 +59,11 @@ class HepMCNodeReader : public SubsysReco
 
   bool PassParticleFilter(HepMC::GenParticle * p);
 
+  ///@some added filter interface for bkg decay study
+  void Set_bkg_mode(){_bkg_mode = true;}// abi
```

Comment:
Could you add a Doxygen comment to explain the meaning of this mode?  The contents are quite complicated, and the function name seems not well explanatory.  You could refer to your document on DocDB.

Reply:
I will do that. Thank you.